iamanager.go

```diff
@@ -191,29 +191,34 @@ func main() {
 
 	identHandler, err := identhandler.New(cfg)
 	if err != nil {
-		log.Fatalf("Can't create ident handler: %s", err)
```

Comment:
IAM should fails if one of its component can't be created.

Reply:
added os.Exit(1)